src/main/java/com/danielagapov/spawn/Controllers/EventController.java

```diff
@@ -70,23 +70,15 @@ public ResponseEntity<List<? extends IEventDTO>> getEventsCreatedByUserId(@PathV
         }
     }
 
-    // TODO: path is wrong
     // full path: /api/v1/events/friendTag/{tagId}
```

Comment:
```suggestion
    // full path: /api/v1/events/friendTag/{friendTagFilterId}
```
keeping consistent with your var name